pages/[id].tsx

```diff
@@ -118,18 +119,18 @@ const CandyMachine: NextPage = () => {
     }
 
     useEffect(() => {
+        setError('')
+        setIsLoading(false)
         ;(async function () {
-            setError('')
-            fetchCandyMachine().then(setCandyMachineConfig)
-            refreshCandyMachineState()
-            await getNfts()
+            const data = await fetchCandyMachine()
+            setCandyMachineConfig(data)
+            // I comment this for now, takes lots of time and cant see anything
```

Comment:
Is the comment still needed?

Reply:
Yes. Takes more than 1 min for me but thats not an issue. In my opinion, the isLoading state should be split between the cmConfig and another state for the nftPart. Because we could show the config while the NFTs load.

Reply:
> Yes. Takes more than 1 min for me but thats not an issue. In my opinion, the isLoading state should be split between the cmConfig and another state for the nftPart. Because we could show the config while the NFTs load.

I changed this in [#149](https://github.com/boxfish-studio/sugar-rush/pull/149)

Reply:
> > Yes. Takes more than 1 min for me but thats not an issue. In my opinion, the isLoading state should be split between the cmConfig and another state for the nftPart. Because we could show the config while the NFTs load.
> 
> I changed this in [#149](https://github.com/boxfish-studio/sugar-rush/pull/149)

Oh sorry buddy, ill uncomment it then!